### PR TITLE
Add some heatpump codes

### PR DESCRIPTION
**Pull request by gerw**

### CODE REVIEW

**Comment by gerw** [comment]
Does anybody know, why `black` is complaining? Has the configuration or the `black´ version changed?

**Comment by Bouni** [comment]
I guess its a version mismatch between the pre-commit-config and the CI

CI:

```
Run pip install black
Collecting black
  Downloading black-24.1.1-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (73 kB)
```

pre-commit
```
repos:
  - repo: https://github.com/psf/black
    rev: 22.12.0
```

File: luxtronik/datatypes.py
Comment (Bouni) [inline luxtronik/datatypes.py:482]: Is this really 8 time the same code?

Comment (gerw) [reply luxtronik/datatypes.py:482]: Yes, this is what you would see in the luxtronik controller for a heat pump of type 31-39. I am not sure, if one can find any of these in the wild...

Comment (Bouni) [reply luxtronik/datatypes.py:482]: Does not sound unfamiliar for AIT to do such a mess 😅 I was just curious if it was kind of a typo/copy and paste error

**Comment by Bouni** [comment]
@gerw If you could review #149 and rebase after the merge I will merge this one

File: luxtronik/datatypes.py
Comment (kbabioch) [reply luxtronik/datatypes.py:482]: What is `ERC`? Does anyone know?

File: luxtronik/datatypes.py
Comment (kbabioch) [inline luxtronik/datatypes.py:515]: Are we sure about this change? Where was this originally coming from?

Comment (gerw) [reply luxtronik/datatypes.py:515]: The `13S` seems to be present in all open source projects reading data from the luxtronik controller. But the luxtronik controller itself would present these heat pumps as `12S`.  Google doesn't find anything related to these strings...

File: luxtronik/datatypes.py
Comment (gerw) [reply luxtronik/datatypes.py:482]: No idea. Maybe this is just some leftover...

**Comment by Bouni** [comment]
@gerw Can you please rebase this one as well?

**Comment by gerw** [comment]
@Bouni: Hurray! :)

Do you know why the bot did not edit the old commit? Is it due to the fact that the old commit was spawned from another work flow? Maybe I just push another time to this branch to see what is happening.

**Comment by gerw** [comment]
Yes, this seems to be the case. The second coverage comment has been edited after the latest push. :partying_face:

File: luxtronik/datatypes.py
Comment (Bouni) [reply luxtronik/datatypes.py:483]: I've looked around and every source I found says something like ERC = unknown. I would merge this and if some day in the future the real meaning comes to light we can fix that.

File: luxtronik/datatypes.py
Comment (Bouni) [reply luxtronik/datatypes.py:516]: I did a quick check against the `appl` binary of the heatpump and there I find only `12S` but n `13S`:

```sh
strings firmware/appl | grep 12S$
MSW_3_12S
MSW12S
MSW3-12S
MSW_3_12S
```

```sh
strings firmware/appl | grep 13S$
```

So I guess `12S` is right

**Review by Bouni — APPROVED** [review]

**Pull request merged**
